src/types.ts

```diff
@@ -35,3 +35,13 @@ export type JSONValue =
   | JSONValue[]
 
 export * from "./types/generated"
+
+export function ensureDefined<T>(
```

Comment:
may be put it in https://github.com/paritytech/opstooling-js/blob/master/src/utils.ts ? 
as I see this is throwable method which acts as a validation during runtime rather than a static type, or I'm missing something?

Reply:
Yeah, fair enough